<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HanziSRS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Hanzi:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QProgressBar" name="hanzi_progress">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Level:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hanzi_level">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hanzi_new">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hanzi_learning">
          <property name="text">
           <string>Learning</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hanzi_review">
          <property name="text">
           <string>Review</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vocabularies:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QProgressBar" name="vocab_progress">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Level:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vocab_level">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vocab_new">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vocab_learning">
          <property name="text">
           <string>Learning</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vocab_review">
          <property name="text">
           <string>Review</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sentences:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QProgressBar" name="sentence_progress">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Level:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sentence_level">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sentence_new">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sentence_learning">
          <property name="text">
           <string>Learning</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sentence_review">
          <property name="text">
           <string>Review</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tags:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tag_input"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="tag_browser">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(233, 233, 233);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionTo_tsv_2"/>
     <addaction name="actionTo_Anki_query_string"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionCategorical_vocab"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionBackup_settings"/>
    <addaction name="actionRestore_backup"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_HanziSRS"/>
    <addaction name="actionAbout_Hanzi_Level_Project"/>
   </widget>
   <widget class="QMenu" name="menuExplore">
    <property name="title">
     <string>Explore</string>
    </property>
    <addaction name="actionVocabulary_dictionary"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionVocabulary"/>
    <addaction name="actionHanzi"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExplore"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionTo_tsv_2">
   <property name="text">
    <string>To *.tsv</string>
   </property>
  </action>
  <action name="actionTo_Anki_query_string">
   <property name="text">
    <string>To Anki query string</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout_HanziSRS">
   <property name="text">
    <string>About HanziSRS</string>
   </property>
  </action>
  <action name="actionCategorical_vocab">
   <property name="text">
    <string>Categorical vocab</string>
   </property>
  </action>
  <action name="actionBackup_settings">
   <property name="text">
    <string>Backup settings</string>
   </property>
  </action>
  <action name="actionRestore_backup">
   <property name="text">
    <string>Restore backup</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionVocabulary_dictionary">
   <property name="text">
    <string>Vocabulary dictionary</string>
   </property>
  </action>
  <action name="actionVocabulary">
   <property name="text">
    <string>Vocabulary</string>
   </property>
  </action>
  <action name="actionHanzi">
   <property name="text">
    <string>Hanzi</string>
   </property>
  </action>
  <action name="actionAbout_Hanzi_Level_Project">
   <property name="text">
    <string>About Hanzi Level Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
